forge-gui/res/cardsfolder/i/infinite_reflection.txt

```diff
@@ -2,13 +2,9 @@ Name:Infinite Reflection
 ManaCost:5 U
 Types:Enchantment Aura
 K:Enchant creature
-A:SP$ Attach | Cost$ 5 U | ValidTgts$ Creature | AILogic$ HighestEvaluation
-T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self+AttachedTo Card | Execute$ RememberInitialAttach | Static$ True
-SVar:RememberInitialAttach:DB$ Pump | RememberObjects$ Valid Card.AttachedBy
-T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self+AttachedTo Card | Execute$ FirstReflections | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME enters the battlefield attached to a creature, each other nontoken creature you control becomes a copy of that creature.
-SVar:FirstReflections:DB$ RepeatEach | RepeatCards$ Creature.nonToken+YouCtrl+IsNotRemembered | UseImprinted$ True | Zone$ Battlefield | RepeatSubAbility$ Reflect | SubAbility$ DBCleanup
-SVar:Reflect:DB$ Clone | Defined$ Remembered | CloneTarget$ Imprinted
-SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ HighestEvaluation
+T:Mode$ ChangesZone | Destination$ Battlefield | ValidCard$ Card.Self+AttachedTo Card | Execute$ TrigClone | TriggerZones$ Battlefield | TriggerDescription$ When CARDNAME enters the battlefield attached to a creature, each other nontoken creature you control becomes a copy of that creature.
+SVar:TrigClone:DB$ Clone | Defined$ Valid Creature.AttachedBy | CloneTarget$ Valid Creature.nonToken+YouCtrl
```

Comment:
this breaks:
> If Infinite Reflection enters the battlefield attached to a creature and then becomes attached to a different creature before its enters-the-battlefield trigger resolves, each other nontoken creature you control will become a copy of the first creature.

Reply:
Ah that explains that static a bit more – thanks

Reply:
I think this is sorted